dashboard/test/lib/pd/payment/payment_calculator_counselor_admin_test.rb

```diff
@@ -6,19 +6,14 @@ class PaymentCalculatorCounselorAdminTest < ActiveSupport::TestCase
       # TIME_CONSTRAINTS_BY_SUBJECT: SUBJECT_ECS_PHASE_4 => {min_days: 2, max_days: 3, max_hours: 18}
       @workshop = create :pd_ended_workshop,
         workshop_type: Pd::Workshop::TYPE_PUBLIC,
-        course: Pd::Workshop::COURSE_ECS,
-        subject: Pd::Workshop::SUBJECT_ECS_PHASE_4,
+        course: Pd::Workshop::COURSE_COUNSELOR,
         num_sessions: 3
```

Comment:
Why did the CounselorAdmin test use an ECS workshop? Your guess is as good as mine. Clearly, this had some unintended consequences.